src/applications/personalization/dashboard/helpers.jsx

```diff
@@ -183,6 +183,12 @@ export function isSIPEnabledForm(savedForm) {
   return true;
 }
 
+// This function is intended to be used as an Array.filter callback
+export function filterOutExpiredForms(savedForm) {
+  const { expiresAt } = savedForm.metadata;
```

Comment:
Do we need to check for a `number` like you do [here](https://github.com/department-of-veterans-affairs/vets-website/pull/16408/files#diff-9e2dc76fded9392ed397624ab97a21b2191bcc5f747756f005277a81b9104c43R192)?

Reply:
No. I'm gonna make sure this is the version that gets merged second

Reply:
sorry, mistyped, but I'm going to make sure the _other_ version with the check is merged second.